changelog.d/sc-547.fixed

```diff
@@ -0,0 +1 @@
+Fixed bug where SSC would generate reachable findings for transitive dependencies, even if a direct dependency at a safe version exists
```

Comment:
not sure if SSC is a user-facing acronym, maybe write Supply Chain? it's also confusing since Semgrep Code is abbreviated SC

Reply:
just switched to Semgrep